extension/httpfs/include/s3fs.hpp

```diff
@@ -12,13 +12,13 @@
 namespace duckdb {
 
 struct S3AuthParams {
-	const std::string region;
-	const std::string access_key_id;
-	const std::string secret_access_key;
-	const std::string session_token;
-	const std::string endpoint;
-	const std::string url_style;
-	const bool use_ssl;
+	std::string region;
```

Comment:
If you include `duckdb/common/string.hpp` we can get rid of all the `std::` here

Reply:
Done!